students/marc_charbo/lesson06/test_mailroom_v4.py

```diff
@@ -9,4 +9,53 @@ def test_run(self):
 
     def test_prompt_user(self):
         mr = mailroom
-        self.assertEqual(mr.prompt_user(),1)
\ No newline at end of file
+        self.assertEqual(mr.prompt_user(),1)
+
+    def test_donor_letter(self):
+        mr = mailroom
+        mr.send_letter()
+        with open('Jim.txt', 'r') as myfile:
+            letter_txt = myfile.read()
+        with open('Jim_test.txt', 'r') as myfile:
+            letter_test = myfile.read()
+
+        self.assertEqual(letter_txt, letter_test)
+
+    def test_quit(self):
+        mr = mailroom
+        self.assertEqual(mr.quit(), 'quit')
+
+    def test_create_report_total(self):
+        donor_list = {'Jim': [25.00, 150.00, 2000.00, 100000.00], 'Linda': [10000.25], 'Bob': [5.03, 100.01, 6.00]}
+        total = 0
+        for key, value in donor_list.items():
+            total = sum(value)
+        self.assertEqual(total, 102175.0)
```

Comment:
Your not referencing the function that actually totals the list there for its not testing your program at all.